packages/common/src/context/commentsContext.tsx

```diff
@@ -73,9 +77,15 @@ export const CommentSectionProvider = ({
       disabled: entityId === 0
     }
   )
+  const { data: currentUserId } = useGetCurrentUserId({})
   const [currentSort, setCurrentSort] = useState<CommentSortMethod>(
     CommentSortMethod.top
   )
+  const dispatch = useDispatch()
+  const playerUid = useSelector(playerSelectors.getUid) ?? undefined
+  const playTrack = useCallback(() => {
+    dispatch(tracksActions.play(playerUid))
```

Comment:
Does this handle the lineup correctly (queues up the 'more by' songs)?

Reply:
really not sure, i just used what was defined in track-page-provider. is that sufficient?

Reply:
Ok I checked on this; looks like this isn't quite what we want - it does queue up the lineup if you're not playing anything but doesn't change songs if you're playing a different song. There is still some more work to do here though so I've got it on my list to revisit this later this week if you want to just leave this in for now; or just leave the method blank for now or something